crates/misc/run-examples/src/main.rs

```diff
@@ -45,10 +49,17 @@ fn main() -> anyhow::Result<()> {
                 .arg(target))?;
         }
         println!("======== Rust example `{}` ============", example);
-        run(Command::new("cargo")
-            .arg("run")
-            .arg("--example")
-            .arg(&example))?;
+        let mut cargo_cmd = Command::new("cargo");
+        cargo_cmd.arg("run").arg("--example").arg(&example);
+        if is_dir {
+            let mut features_path = std::path::PathBuf::from(example_path);
+            features_path.push("CARGO_FEATURES");
```

Comment:
We probably don't need to be so general here, could this detect maybe "async" in the name and pass a hardcoded feature?

(mostly trying to keep the examples dir relatively clean)

Reply:
Sure, I could do that. I figured that putting the features in a file would be more discoverable for someone trying to run it standalone than detecting a part of the name.